docs/spec/v1beta2/kustomization.md

```diff
@@ -1365,9 +1365,9 @@ them on the cluster.
 For secrets in `.json`, `.yaml` and `.env` format, make sure you specify the input type when encrypting them with SOPS:
 
 ```sh
-cat config.json | sops -e --input-type=json > config.json.encrypted
-cat config.yaml | sops -e --input-type=yaml > config.yaml.encrypted
-cat config.env | sops -e --input-type=env > config.env.encrypted
+sops -e --input-type=json /dev/stdin < config.json > config.json.encrypted
+sops -e --input-type=yaml /dev/stdin < config.yaml > config.yaml.encrypted
+sops -e --input-type=env /dev/stdin < config.env > config.env.encrypted
```

Comment:
Adding `/dev/stdin` was necesary for me to make the commands work (tried it with SOPS 3.7.1 and 3.7.2).
Dropping the `cat` was meant to simplify the command. I can put it back in if someone prefers the old version.